chat/src/main/resources/application.yml

```
@@ -82,3 +85,5 @@ feign:
         readTimeout: 5000 # 읽기 타임아웃 (ms)
         retryer: feign.Retryer.Default # 재시도 정책
 
+slack:
+  web-hook-url: ${SLACK_WEBHOOK_URL}
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)